tutormfe/patches/local-docker-compose-dev-services

```diff
@@ -6,7 +6,7 @@
         context: ../plugins/mfe/build/mfe/
         target: {{ app["name"] }}-dev
     ports:
-        - "127.0.0.1:{{ app["port"] }}:{{ app["port"] }}"
+        - "{{ app["port"] }}:{{ app["port"] }}"
```

Comment:
Unfortunately, this is not really a great idea because it exposes the app port to everyone. (0.0.0.0)

That being said, I now realize that the dev ports are indeed open to everyone for the core Tutor app... https://github.com/overhangio/tutor/blob/master/tutor/templates/dev/docker-compose.yml So I guess it's fine to do the same for the MFE.